src/css/components/_live.scss

```diff
@@ -12,3 +11,55 @@
   width: auto;
   text-align: left;
 }
+
+// hide the LiveDisplay when not live or when
+// the new liveui is in use
+.video-js:not(.vjs-live) .vjs-live-control,
+.video-js.vjs-liveui .vjs-live-control {
+  display: none;
+}
+
+// css for the new live ui below
+.video-js .vjs-seek-to-live-control {
+  cursor: pointer;
+  @include flex(none);
+  display: inline-flex;
+  height: 100%;
+  margin-left: 0.7em;
+  font-size: 1em;
+  line-height: 3em;
+  width: auto;
+}
+
+.vjs-no-flex .vjs-seek-to-live-control {
+  display: table-cell;
+  width: auto;
+  text-align: left;
+}
+
+// hide the SeekToLive button when not live and
+// when the liveui is not in use
+.video-js.vjs-live:not(.vjs-liveui) .vjs-seek-to-live-control,
+.video-js:not(.vjs-live) .vjs-seek-to-live-control {
+  display: none;
+}
+
+// only show as a pointer when we will seek to live edge
+.vjs-seek-to-live-control.vjs-control.vjs-at-live-edge {
+  cursor: auto;
+}
+
+.vjs-seek-to-live-control .vjs-seek-to-live-circle {
+  border-radius: 50%;
+  width: 0.75em;
+  height: 0.75em;
+  margin-top: auto;
+  margin-bottom: auto;
+  margin-right: 0.3em;
+  background: radial-gradient(#808080, #666666);
```

Comment:
I think going with a flat color might be better, like `#888` here and `red` below